src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/VacationRequestsTest.kt

```diff
@@ -42,6 +42,7 @@ class VacationRequestsTest: AbstractTest() {
     days = 2,
     type = VacationType.VACATION,
     message = "Lomaa!!!",
+    draft = false,
```

Comment:
Add tests for filtering listing with the `draft` parameter.

Reply:
Resolved